avocado/utils/memory.py

```diff
@@ -48,6 +48,13 @@ def memtotal():
     return read_from_meminfo('MemTotal')
 
 
+def availablememtotal():
+    """
+    Read ``MemAvailable`` from meminfo.
+    """
+    return read_from_meminfo('MemAvailable')
```

Comment:
TBH I didn't noticed we have the `freememtotal` and `memtotal` but I don't think such methods should be present. The difference in test code is:

```python
memory.read_from_meminfo("MemAvailable")
```
vs.
```python
memory.availablememtotal()
```

Which is almost the same size but requires 7 lines in library. I'd suggest improving the docstring in `read_from_meminfo` by listing:

```python
    """
    ...
    Most important keys:
    MemTotal - Overall memory size
    MemAvailable - Currently available memory
    MemFree - Free memory
    ...
    """
```

and skip those rather simplistic methods.